File: pset6/mario/less/mario.py
from cs50 import get_int


while True:

    n = get_int('Height: ')

    if n >= 1 and n <= 8:
     break
    
    print('Height ivalid.')

for i  in range(n):
    print(' '*(n-1-i), end='')
    print('#'*(1+i))

File: mario1/mario.c
#include <stdio.h>
#include <cs50.h>

int main(void)
{
    //  Pega o valor de n com o usuário
    int n;
    do
    {
        n = get_int("Digite de 1 a 8: ");
    }
    while (n > 8 || n < 1);

    for (int i = 0; i < n; i++)
    {
        for (int e = 0; e < (n - i); e++)
        {
            printf(" ");
        }
        for (int j = 0; j <= i; j++)
        {
            printf("#");
        }
        printf("\n");
    }
}